app/views/MultipleQuestionView.scala.html

```diff
@@ -154,3 +157,10 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-3">@renderMessage(viewModel.hea
         )
     )).withFormField(form(s"value.${index + 1}")))
 }
+
+@errorLinkOverrideHelper = @{
+  viewModel.page.fields.zipWithIndex.collect {
+    case (field @ QuestionField(_, _, _, _, FieldType.Date), index) =>
```

Comment:
Do you need a default case here to cater for anything that is not QuestionField with FieldType.Date?

Reply:
I think no, because this is an override, for other type of fields the error links doesn't need an overide.